src/services/__tests__/dealer.test.ts

```diff
@@ -13,32 +13,32 @@ describe("Dealer", () => {
   describe("#fetchDealerSuggestions", () => {
     it("encodes the query", async () => {
       const query = "k+k"
-      await fetchDealerSuggestions(query)
+      await fetchDealerSuggestions({ query })
 
       expect(fetch).toHaveBeenCalledWith(
-        `dealer.service.test/dealers/suggestions?q=${encodeURIComponent(
-          query
-        )}`,
+        expect.stringContaining(
+          `dealers/suggestions?q=${encodeURIComponent(query)}`
+        ),
         expect.any(Object)
       )
     })
   })
 
   describe("#fetchDealer", () => {
     it("accepts language option", async () => {
-      await fetchDealer(123, { language: "de" })
+      await fetchDealer({ id: 123, language: "de" })
 
       expect(fetch).toHaveBeenCalledWith(
-        `dealer.service.test/dealers/123?language=de`,
+        expect.stringContaining("dealers/123?language=de"),
         expect.any(Object)
       )
     })
 
     it("works without the language option", async () => {
-      await fetchDealer(123)
+      await fetchDealer({ id: 123 })
```

Comment:
it might be a bit arbitrary so up to you but since you are on it it might be helpful to name the id params like `dealerId` and `listingsId` (we might have other ids) so it would be more obvious in the consumer code what ids are passed around

Reply:
Just saw below that you already named the other ones so it probably make sense to be consistent

Reply:
Yeah thought about that too. Will do 👍